rust/src/python/libnmstate/clib_wrapper.py

```diff
@@ -111,6 +111,9 @@ def apply_net_state(
     if not commit:
         flags |= NMSTATE_FLAG_NO_COMMIT
 
+    if not save_to_disk:
+        flags |= NMSTATE_FLAG_MEMORY_ONLY
+
```

Comment:
Could you extend this flag on clib and golang bindings too? I think it is not present there and it should be.. thanks!